playwright.config.ts

```diff
@@ -31,8 +31,22 @@ export default defineConfig({
   },
   projects: [
     {
-      name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
+      name: 'chromium desktop',
+      testIgnore: /.*mobile.spec.ts/,
+      use: {
+        ...devices['Desktop Chrome'],
+        viewport: { width: 1920, height: 1080 },
+        isMobile: false,
+      },
+    },
+    {
+      name: 'chromium mobile',
+      testIgnore: /.*desktop.spec.ts/,
+      use: {
+        ...devices['Desktop Chrome'],
```

Comment:
Should this be `...devices['Mobile Chrome'].` ?

Reply:
We could select [a specific mobile device](https://playwright.dev/docs/api/class-playwright#playwright-devices), but I opted to just stick with `Desktop Chrome` and set the viewport based on what we use in designs instead.